repo/packages/N/namerd/2/marathon.json.mustache

```diff
@@ -0,0 +1,61 @@
+{
+  "id": "{{namerd.name}}",
+  "instances": {{namerd.instances}},
+  "cpus": {{namerd.cpus}},
+  "mem": {{namerd.mem}},
+  "acceptedResourceRoles": [ "{{namerd.resource-role}}" ],
+  "constraints": [["hostname", "UNIQUE"]],
+  "maintainer": "hello@buoyant.io",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.namerd-docker}}",
+      "network": "BRIDGE",
+      "privileged": true,
+      "portMappings": [
+        {
+          "containerPort": {{namerd.admin-port}},
+          "hostPort": {{namerd.admin-port}},
+          "servicePort": 0,
+          "protocol": "tcp"
+        },
+        {
+          "containerPort": {{namerd.http-port}},
+          "hostPort": {{namerd.http-port}},
+          "servicePort": 0,
+          "protocol": "tcp"
+        },
+        {
+          "containerPort": {{namerd.thrift-port}},
+          "hostPort": {{namerd.thrift-port}},
+          "servicePort": 0,
+          "protocol": "tcp"
+        }
+      ]
+    }
+  },
+  "healthChecks": [
+    {
+      "protocol": "HTTP",
+      "portIndex": 0,
+      "path": "/admin/ping"
+    }
+  ],
+  "args": [
+    "/mnt/mesos/sandbox/{{namerd.config-filename}}"
+  ],
+  "requirePorts":true,
+  "ports": [
+    {{namerd.admin-port}},
+    {{namerd.http-port}},
+    {{namerd.thrift-port}}
+  ],
+  "uris": [
+    "{{namerd.config-uri}}/{{namerd.config-filename}}"
```

Comment:
We should express this in the `resource.json` file. See https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/C/cassandra/1/resource.json#L4-L6 and https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/C/cassandra/1/marathon.json.mustache#L10-L13